configs/config-default.yaml

```
@@ -0,0 +1,28 @@
+hyperparameters:
+  dataset_name: "roszcz/ecg-segmentation-ltafdb" # huggingface dataset
+  batch_size: 256
+  num_workers: 8
+  lr: 3e-4
+  weight_decay: 0.01
+  num_epochs: 10
+  accelerator: "gpu"
+  precision: 16
+  overfit_batches: 0
+  dataset_split_seed: 0 # seed to make random_split deterministic
```

```suggestion
train:
  dataset_name: "roszcz/ecg-segmentation-ltafdb" # huggingface dataset
  batch_size: 256
  num_workers: 8
  lr: 3e-4
  weight_decay: 0.01
  num_epochs: 10
  accelerator: "gpu"
  precision: 16
  overfit_batches: 0
  dataset_split_seed: 0 # seed to make random_split deterministic
```

I think `hyperparameters` is not descriptive enough here and `train` seem like a better config section name. 

Notice that the `unet` params are also technically hyperparameters, but if you include that section here almost every config position would be in that section :)